Integrations/BitDam/BitDam.py

```diff
@@ -0,0 +1,234 @@
+    returned_sha1 = parse_scan_file_response(response)
+    # Build demisto reponse
+    response_content = {'SHA1': returned_sha1}
+    response_context = {'BitDam': {'FileScan': {'SHA1': returned_sha1}}}
+    return build_json_response(response_content, response_context, "File was submitted successfully")
+
+
+def scan_file_command():
+    # Get data to build the request
+    entry_id = demisto.args().get('entryId')
+    file_name = get_file_name(entry_id)
+    file_bytes = get_file_bytes(entry_id)
+    json_data = {'file_name': file_name,
+                 'file_data_base64': base64.b64encode(file_bytes)}
+    raw_json = json.dumps(json_data, ensure_ascii=False)
+    url = "{}{}".format(get_url_base_with_trailing_slash(), SCAN_FILE_URL)
+    auth_headers = get_auth_headers()
+
+    # Send the HTTP request
+    response = requests.post(url, data=raw_json, headers=auth_headers, verify=not UNSECURE)
+    return response
+
+
+def parse_scan_file_response(response):
+    # Parse response
+    if RESPONSE_CODE_OK != response.status_code:
+        raise Exception("Scan file failed. Response code -{}, Data- '{}'".format(str(response.status_code), response.content))
+    response_json = json.loads(response.content)
+    if 'sha1' not in response_json:
+        raise Exception(
+            "Scan file failed. Bad response json - {}".format(response.content))
+    returned_sha1 = response_json['sha1']
+    return returned_sha1
+
+
+def get_file_verdict():
+    identifier_value = demisto.args().get('idValue')
+    response = get_file_verdict_command(identifier_value)
+    verdict, status = parse_get_file_verdict_response(response)
+    response_content = {'STATUS': status,
+                        'VERDICT': verdict,
+                        'ID': identifier_value}
+    context = {}
+    context['BitDam.Analysis(val.ID && val.ID == obj.ID)'] = {
+        'Status': status,
+        'Verdict': verdict,
+        'ID': identifier_value
+    }
+
+    if VERDICT_MALICIOUS == verdict:
+        context[outputPaths['file']] = {'SHA1': identifier_value}
+        context[outputPaths['file']]['Malicious'] = {
+            'Vendor': 'BitDam',
+            'Description': 'Process whitelist inconsistency by bitdam-get-file-verdict',
+            'Name': identifier_value
+        }
+
+    dbotscore = verdict_to_dbotscore(verdict)
+    if dbotscore:
+        context[outputPaths['dbotscore']] = {
+            'Indicator': identifier_value,
+            'Type': 'File',
+                    'Vendor': 'BitDam',
+                    'Score': dbotscore
+        }
+    response_context = context
+    return build_json_response(response_content, response_context,
+                               "Get file verdict was performed successfully")
+
+
+def parse_get_file_verdict_response(response):
+    # Parse results
+    if RESPONSE_CODE_OK != response.status_code:
+        raise Exception("Get file verdict failed. Response code -{}, Data- '{}'".format(str(response.status_code),
```

Comment:
should add try except in end of integration, catch the exception and return_error